cli/tests/pcluster/cli/test_list_cluster_log_streams.py

```diff
@@ -123,6 +130,7 @@ def test_execute(self, mocker, set_env, args):
     def _build_cli_args(args):
         cli_args = []
         for k, val in args.items():
-            val = " ".join(val) if k == "filters" else val
-            cli_args.extend([f"--{to_kebab_case(k)}", val])
+            val = val if k == "filters" else [val]
+            cli_args.extend([f"--{to_kebab_case(k)}"])
```

Comment:
Since it's only an element, it would be cleaner to do `cli_args.append(f"--{to_kebab_case(k)}")` (you don't have to build a new list).

Reply:
Thanks, this makes sense, will fix